controllers/boardgamesController.js

```diff
@@ -0,0 +1,78 @@
+
+
+var db = require('../models');
+
+
+// function for initial page render
+function index (req, res) {
+  db.Boardgame.find({}, function (err, allGames) {
+    if (err) {
+      console.log('error from controller index', err);
+    }
+    res.json(allGames);
+  })
+}
+
+// Creates new board game
+function create (req, res) {
+  db.Boardgame.create(req.body, function(err, game) {
+    if (err) {
+      console.log('ERROR from controller create', err);
+    }
+    if (game.image) {
```

Comment:
Could think about doing this before create, so that you only have a single database transaction:
```js
if (!req.body.image) {
  req.body.image = 'images/generic-game-image.jpg';
}
db.Boardgame.create(...
```